### PR TITLE
Dbp 281 workflow image push to main

### PR DESCRIPTION
**Pull request by maxi418**
# Description

## Approval for review

- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: .github/workflows/check-nest-lint.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        type: string
+        description: "Version to use for node"
+        required: true
+      deploy_stage:
+        type: string
+        description: "Deployment stage"
+        required: true
+env:
+  NODE_VERSION: ${{ inputs.node_version }}
+  DEPLOY_STAGE: ${{ inputs.deploy_stage }}
```
Comment (simoncolincap) [inline .github/workflows/check-nest-lint.yaml:16]: Is the deploy_stage variable/env used anywhere?

Comment (maxi418) [reply .github/workflows/check-nest-lint.yaml:16]: The variable is needed when running the tests, if not set an error is thrown.

Comment (maxi418) [reply .github/workflows/check-nest-lint.yaml:16]: Tested here: https://github.com/dBildungsplattform/dbildungs-iam-server/actions/runs/6574557741/job/17859846102

Comment (aimee-889) [reply .github/workflows/check-nest-lint.yaml:16]: nice to know that everyone is wondering about the same thing :D

**Pull request merged**